airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/auth/AuthButton.test.tsx

```diff
@@ -114,13 +118,14 @@ describe("auth button", () => {
     expect(successMessage).toBeInTheDocument();
   });
 
-  it("if authError is true, it renders the correct message", () => {
+  it("renders an error if there are any auth fields with empty values", () => {
```

Comment:
Are there any other types of errors we might encounter for this?  It looks like the button only actually accounts for missing credentials and no other errors, but I wonder if we want to expand that?